### PR TITLE
Task-48-For-Testing

### PR DESCRIPTION
**Pull request by hnbmunoz**

### CODE REVIEW

File: react-client/src/Components/App.js [hunk 1]
```diff
@@ -345,7 +372,10 @@ function App() {
 
       {/* website page renders here... */}
       {!PRODUCTION_MODE && <div id="samplePage"></div>}
-      <div id="dimmer" className={`${focusMode && 'dimmer-show'}`}></div>
+      {/* <div id="dimmer" className={`${focusMode && 'dimmer-show'}`}></div> */}
```
Comment (reciosonny) [inline react-client/src/Components/App.js]: Remove comment

File: react-client/styles/_base.scss [hunk 2]
```diff
@@ -57,9 +61,10 @@ $DialogDetailPadding: 10px 0px 0px 0px;
 
 
 .focused-targeted-element {
-  z-index: 9999;
-  position: relative;
-  background: #FFFFFF;
+  z-index: 9999 !important;  
+  box-shadow: 999vmax 999vmax  999vmax 9999vmax rgba(0, 0, 0, 0.95);
```
Comment (reciosonny) [inline react-client/styles/_base.scss:65]: Why do we use vmax here?

**Comment by reciosonny** [comment]
Other than my comments I think this looks good.

File: react-client/src/Components/App.js [hunk 3]
```diff
@@ -292,20 +292,27 @@ function App() {
...
-      currentEl.classList.remove(currDomInfo.cssClassesAssigned);
-
-      setDomInfo(newDomInfo);
-      document.querySelector(".focused-targeted-element").style.visibility =  "hidden";
-      document.querySelector(".focused-element").classList.remove("focused-element");
-      setFocusMode(false);
-    }  
+    if (focusMode === true){ 
+      const focusedEl = document.querySelector(".focused-element");
+      if (focusedEl === currentEl){
```
Comment (reciosonny) [inline react-client/src/Components/App.js]: Fix spacing

File: react-client/src/Components/App.js [hunk 4]
```diff
@@ -292,20 +292,27 @@ function App() {
...
 
-    if(currentEl) 
-      currentEl.classList.remove(currDomInfo.cssClassesAssigned);
-
-      setDomInfo(newDomInfo);
-      document.querySelector(".focused-targeted-element").style.visibility =  "hidden";
-      document.querySelector(".focused-element").classList.remove("focused-element");
-      setFocusMode(false);
-    }  
+    if (focusMode === true){
```
Comment (reciosonny) [inline react-client/src/Components/App.js]: Fix spacing

File: react-client/src/Components/App.js [hunk 5]
```diff
@@ -292,20 +292,27 @@ function App() {
...
+      if (focusedEl === currentEl){   
+        if(currentEl) 
+          currentEl.classList.remove(currDomInfo.cssClassesAssigned);  
+          setDomInfo(newDomInfo);            
+          document.querySelector(".focused-targeted-element").style.visibility =  "hidden";
+          document.querySelector(".focused-element").classList.remove("focused-element");          
+          setFocusMode(false);
+      }
+    } 
+    else{
```
Comment (reciosonny) [inline react-client/src/Components/App.js]: Fix spacing

File: react-client/styles/_base.scss (hunk 2 above)
Comment (hnbmunoz) [reply react-client/styles/_base.scss:65]: Vmax represents the size that the shadow would cover actually enough nmn  0 0 0 999 vmax pro gn oversize ko n lng just to cover a big space

**Review by reciosonny — APPROVED** [review]

**Pull request merged**